Oleksii_Fedorchuk/HW18_Oleksii_Fedorchuk/HW18_Oleksii_Fedorchuk/test_human.py

```diff
@@ -0,0 +1,102 @@
+import pytest
+
+
+@pytest.mark.parametrize("name, age, gender", [("Harry", 10, "male")], ids=["Human is created"])
+def test_human_creating(human_template, name, age, gender):
+    """
+    Pre-Condition:
+    create_human texture is created
+    Steps:
+    1. Add name
+    2. Add age
+    3. Add gender
+    ER: [Human is created]
+    """
+    human = human_template(name, age, gender)
+    assert human
```

Comment:
you should compare AR(actual result) with ER(expected result)

Reply:
Will be fixed in an update